routingpy/routers/osrm.py

```diff
@@ -115,6 +115,12 @@ def directions(
             from in order of visit.
         :type locations: list of list
 
+        :param profile: Optionally specifies the mode of transport to use when calculating
+            directions. Note that this strongly depends on how the OSRM server works. E.g.
+            the FOSSGIS instance expects 'driving' and chose to encode the 'profile' in the
```

Comment:
It's not quite true that the FOSSGIS instance expects `driving`, any combination of letters and characters works here and produces the same result, e.g. [https://routing.openstreetmap.de/routed-car/route/v1/whatever/5.120261,52.094867;5.124008,52.081887;5.124723,52.086975](https://routing.openstreetmap.de/routed-car/route/v1/whatever/5.120261,52.094867;5.124008,52.081887;5.124723,52.086975)

So maybe point out that there simply is no effect when using this public instance

Reply:
yeah of course..